godotproject/Menus/ThanksForPlayingScreen/ThanksForPlayingScreen.gd

```diff
@@ -0,0 +1,10 @@
+extends MarginContainer
+
```

Comment:
Has this been linked to the end of The Big City level?

Reply:
Yep

Reply:
nice!